package.json

```diff
@@ -53,8 +53,10 @@
     "mocha": "^9.2.2",
     "nodemon": "2.0.19",
     "prettier": "^2.6.2",
+    "proxyquire": "^2.1.3",
     "sinon": "^11.1.2",
     "telegram-test-api": "^2.5.0",
-    "typescript": "^5.1.6"
+    "typegram": "^5.2.0",
```

Comment:
@danfercf1 why was this new dependency added?

Reply:
That dependency is imported in start.ts file, and it wasn't added in the package.json file, the bot works because the package-lock.json file has the dependency but it's not a good practice to not have synchronized both files

Reply:
ok, next time split that change in a different PR please